test/integration/narrow/app/import/access-db.test.js

```diff
@@ -0,0 +1,19 @@
+jest.mock('read-excel-file/node')
```

Comment:
These tests could do with being in `local` rather than `narrow`

Reply:
Moved